ocaml/idl/datamodel_errors.ml

```diff
@@ -107,6 +107,10 @@ let _ =
     ~doc:"A VDI with the specified location already exists within the SR" ();
   error Api_errors.memory_constraint_violation ["constraint"]
     ~doc:"The dynamic memory range does not satisfy the following constraint." ();
+  error Api_errors.memory_constraint_violation_order ["constraint"]
+    ~doc:"The dynamic memory range violates static_min <= dynamic_min <= dynamic_max <= static_max." ();
```

Comment:
I would leave the term constraint in: `The dynamic memory range violates the constraint static_min <= dynamic_min <= dynamic_max <= static_max.` Same below.